notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -311,7 +316,8 @@ internal class NotifiqueListView(
   }
 }
 
-private val timestampFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US)
+private val dateTimestampFormat = SimpleDateFormat("MMM d h:mm a", Locale.US)
+private val timestampFormat = SimpleDateFormat("h:mm a", Locale.US)
```

Comment:
Does this respect the user's 24-hour clock system time preference?

Reply:
Probably not, ill update